koku/koku/middleware.py

```diff
@@ -64,7 +64,7 @@
 
 def is_no_auth(request):
     """Check condition for needing to authenticate the user."""
-    no_auth_list = ["/status", "openapi.json", "/metrics"]
+    no_auth_list = ["/status", "openapi.json", "/metrics", "/recommendations/openshift/openapi.json"]
```

Comment:
@chambridge I may be wrong but looking at the nginx config - [here](https://github.com/project-koku/koku/blob/main/deploy/clowdapp.yaml#L4311-L4317)  I think for ROS API `/recommendations/openshift/openapi.json` koku middleware will not get triggered because request will be directly forwarded to ROS service by nginx. Thought?